### PR TITLE
Misc UXA fixes for 12279

### PR DESCRIPTION
**Pull request by owillborn**
*Issue #:* /bcgov/entity#12279

*Description of changes:*
- Icon sizing changes
- Field padding fix
- Text changes in registration submission failed modal

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by doug-lovett — APPROVED** [review]
Straightforward, looks good.

File: ppr-ui/src/components/mhrRegistration/ReviewConfirm/HomeLocationReview.vue [hunk 1]
```diff
@@ -334,4 +334,9 @@ export default defineComponent({
 .error-text {
   font-size: 16px;
 }
+
+.home-location-icon {
```
Comment (dimak1) [inline ppr-ui/src/components/mhrRegistration/ReviewConfirm/HomeLocationReview.vue]: If the icon size is the same across the app, maybe we could put it into the base.scss?

**Review by dimak1 — APPROVED** [review]
Approved with a comment.

File: ppr-ui/src/components/mhrRegistration/ReviewConfirm/HomeLocationReview.vue (hunk 1 above)
Comment (owillborn) [reply ppr-ui/src/components/mhrRegistration/ReviewConfirm/HomeLocationReview.vue]: good call I'll do that

File: ppr-ui/src/components/mhrTransfers/DeathCertificate.vue [hunk 2]
```diff
@@ -51,7 +51,7 @@
         </v-row>
         <v-row>
           <v-spacer></v-spacer>
-          <v-col cols="9" class="pl-3">
+          <v-col cols="9" class="pl-1">
```
Comment (cameron-eyds) [inline ppr-ui/src/components/mhrTransfers/DeathCertificate.vue:54]: Does the 8px adjustment still keep it in alignment with the Land Lease or Ownership Checkbox?

Screen Shot 2023-03-21 at 5 11 09 PM

**Review by cameron-eyds — APPROVED** [review]
Looks fine but might want to confirm with Sara that adjusting that spacing will put it out of alignment with the Land Lease Checkbox, from what i can tell.

File: ppr-ui/src/components/mhrTransfers/DeathCertificate.vue (hunk 2 above)
Comment (owillborn) [reply ppr-ui/src/components/mhrTransfers/DeathCertificate.vue:54]: Yes it stays in line

Comment (cameron-eyds) [reply ppr-ui/src/components/mhrTransfers/DeathCertificate.vue:54]: What is this voodoo that keeps it in alignment, unless the land lease is moved too?

Comment (owillborn) [reply ppr-ui/src/components/mhrTransfers/DeathCertificate.vue:54]: The voodoo is no longer voodooing. Will confer with Sara for what to do with the land lease checkbox

Comment (owillborn) [reply ppr-ui/src/components/mhrTransfers/DeathCertificate.vue:54]: Moved Land Lease checkbox in line

Comment (cameron-eyds) [reply ppr-ui/src/components/mhrTransfers/DeathCertificate.vue:54]: hahahah right on man, sounds good 👍

**Pull request merged**